hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -112,6 +112,7 @@ hedera.transaction.maxMemoUtf8Bytes=100
 hedera.transaction.maxValidDuration=180
 hedera.transaction.minValidDuration=15
 hedera.transaction.minValidityBufferSecs=10
+hedera.transaction.eip2930.enabled=false
```

Comment:
So just confirming: We're not enabling this yet for real, but it _is_ enabled for testing.

I'm not sure but this _might_ deserve a comment up at `hedera-app/src/test/resources/bootstrap.properties` just to note the difference in behavior tests vs production (at this time).

Reply:
We don't want to accidentally break mirror node before it's ready so we're going with off by default now and then enable by default in the next alpha version.
If `release/0.41` branch has already occurred we can actually set this to true and make sure it's set to false in 0.4q until we've confirmed E2E